pyomo/solvers/plugins/solvers/cplex_direct.py

```diff
@@ -105,13 +105,16 @@ def _apply_solver(self):
             for block in self._pyomo_model.block_data_objects(descend_into=True, active=True):
                 for var in block.component_data_objects(ctype=pyomo.core.base.var.Var, descend_into=False, active=True, sort=False):
                     var.stale = True
+        _log_file = self._log_file
+        if (self.version()[0] > 12) or (self.version()[0] == 12 and self.version()[1] >= 10):
```

Comment:
Can we just do `self.version()[:2] >= (12,10)` for clarity?

Reply:
You shouldn't even need the `[:2]` if that's cleaner 😉 

```python
assert (12, 7, 1) < (12, 10)
```